web/src/engine/build.sh

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bash
+#
+# Compile keymanweb and copy compiled javascript and resources to output/embedded folder
+#
+
+# set -x
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+WORKING_DIRECTORY=`pwd`
+
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
+
+# ################################ Main script ################################
+
+# Definition of global compile constants
+
+MAIN="engine/main" # Covers all engine code, including submodules like those listed below.
+DEVICEDETECT="engine/device-detect"
+ELEMENTWRAPPERS="engine/element-wrappers"
+
+BUILD_BASE="build"
+
+OUTPUT_DIR="obj"
```

Comment:
Given these strings have no spaces, technically none of them need quotes :grin:
```suggestion
MAIN=engine/main     # Covers all engine code, including submodules like those listed below.
DEVICEDETECT=engine/device-detect
ELEMENTWRAPPERS=engine/element-wrappers

BUILD_BASE=build

OUTPUT_DIR=obj
```